routes/api.php

```diff
@@ -121,6 +121,8 @@
         Route::get('users/{user}/membership', 'UserMembershipController@show')->name('membership.show');
 
         Route::get('h5p/settings', 'H5pController@create');
+        Route::get('suborganization/{suborganization}/projects/{project}/offline-project', 'ProjectDownloadController@exportProject');
```

Comment:
Wo follow REST API standards for naming the URL. Please make route as:
'suborganization/{suborganization}/projects/{project}/export/offline'